app/services/by_trust_project_fetcher_service.rb

```diff
@@ -0,0 +1,42 @@
+class ByTrustProjectFetcherService
+  def call
+    conversion_counts = conversion_count_by_trust
+
+    if conversion_counts
+      trusts = get_trusts(conversion_counts.keys)
+
+      sort_trust_objects_by_name(build_trust_objects(trusts, conversion_counts))
+    else
+      []
+    end
+  end
+
+  private def conversion_count_by_trust
+    projects = Project.not_completed
```

Comment:
I'd make this `Conversion::Project`, just so we don't have to circle back later when we start using Transfer projects.

Reply:
Nice!! 👍